pkg/sentry/platform/ring0/lib_arm64.s

```diff
@@ -15,6 +15,13 @@
 #include "funcdata.h"
 #include "textflag.h"
 
+TEXT ·FlushTlbAll(SB),NOSPLIT,$0
+	DSB $10
+	WORD $0xd508831f
```

Comment:
Can you add a comment indicating what instruction this encodes? (IIUC it's TLBI VMALLE1IS)

Reply:
Done.